package.json

```diff
@@ -43,7 +43,7 @@
     "@google-cloud/logging-bunyan": "^0.7.0",
     "chalk": "^2.3.1",
     "commander": "^2.14.1",
-    "github": "^13.1.0",
```

Comment:
eehm did you read the release notes in the comment? This will break todo, you have to migrate :)

Reply:
I did but I didn't realize you'd removed the package entirely 😲